lib/neovim-plugin.nix

```diff
@@ -48,9 +48,12 @@ with lib;
       extraPackages ? [ ],
       callSetup ? true,
       installPackage ? true,
+      allowLazyLoad ? true,
```

Comment:
If the current implementation depends on `callSetup`, maybe do `allowLazyLoad ? callSetup`?

Reply:
Even if plugins do not require a setup function, they are still lazy-loaded by lz.n. Their modules are not loaded until the conditions are reached.